okhttp/src/main/java/com/squareup/okhttp/ConnectionSpec.java

```diff
@@ -242,7 +242,7 @@ public Builder cipherSuites(CipherSuite... cipherSuites) {
       return cipherSuites(strings);
     }
 
-    Builder cipherSuites(String[] cipherSuites) {
+    public Builder cipherSuites(String[] cipherSuites) {
```

Comment:
I'm happy either way, the most important thing is being able to do this.

Would you prefer varargs for a public API?